app/models/project.rb

```diff
@@ -115,7 +120,11 @@ def organization_name
   end
 
   def jira_key
-    jira_project&.jira_project_key
+    product&.jira_project&.jira_project_key
+  end
+
+  def jira_project?
```

Comment:
Is this being used?

Reply:
Yes, when creating developer metrics chartkick. If the project has a jira project it adds the metrics related.